src/components/RedocStandalone.tsx

```diff
@@ -43,6 +45,10 @@ export class RedocStandalone extends React.PureComponent<RedocStandaloneProps> {
 
     const normalizedOpts = new RedocNormalizedOptions(options);
 
+    if (normalizedOpts.nonce !== undefined) {
+      __webpack_nonce__ = normalizedOpts.nonce;
```

Comment:
It may crash with `ReferenceError` in non-webpack env, e.g. when used by redoc-cli. We should add a check if the `__webpack_nonce__` is defined.

Reply:
I had to use `try/catch` because `__webpack_nonce__` is initially set to `undefined` making disambiguation between variable being declared but not set and not being declared at all very hard.

If you have a better way, I'm open to suggestion.